java/src/main/java/ai/rapids/cudf/Cuda.java

```diff
@@ -39,7 +43,7 @@
     protected boolean cleanImpl(boolean logErrorIfNotClean) {
       boolean neededCleanup = false;
       long origAddress = stream;
-      if (stream != 0) {
+      if (stream != CUDA_STREAM_DEFAULT && stream != CUDA_STREAM_LEGACY && stream != CUDA_STREAM_PER_THREAD) {
```

Comment:
nit, could break this line down into multiple lines for readability.

Reply:
Done.